tools/frrcommon.sh.in

```diff
@@ -150,6 +150,7 @@ daemon_prep() {
 daemon_start() {
 	local dmninst daemon inst args instopt wrap bin
 	daemon_inst "$1"
+	all "$2"
```

Comment:
Is this supposed to be an assignment ?

Reply:
yes. that is what I believe the syntax is.

Reply:
looks like a function call syntax to me, it should be

```
all=$2
```